docs/configuration.md

```diff
@@ -1365,6 +1366,13 @@ Apart from these, the following properties are also available, and may be useful
         </td>
     </tr>
     <tr>
+        <td><code>spark.ssl.keyStoreType</code></td>
+        <td>JKS</td>
+        <td>
+            The type of the key-store.
```

Comment:
"key-store" is not really correct; if you feel like correcting the other ones, that's fine with me.